lgc/interface/lgc/Builder.h

```diff
@@ -760,8 +760,10 @@ class Builder : public BuilderBase {
   // Create a buffer length query based on the specified descriptor.
   //
   // @param bufferDesc : The buffer descriptor to query.
+  // @param offset: The byte offset of the variable array. It is clamped at 0 if robustAccess is on.
```

Comment:
I think we should not be documenting this in terms of a SPIR-V variable array. Something like "The byte offset to subtract from the buffer length. If allowNullBufferDesc is on, the result is clamped at 0 if the offset is greater than the buffer length"

Reply:
My English grammar was not great there. :-)

"The result is clamped at 0 if the offset is greater than the buffer length and allowNullBufferDesc is on."

Reply:
Fixed.

Reply:
I don't think the comment is fixed is it?